frontend/src/Pages/LessonAnswerPage.tsx

```diff
@@ -0,0 +1,68 @@
+import ChoicesButton from '../components/common/ChoicesButton';
+import Done from '../components/common/Done';
+import Navbar from '../components/common/Navbar';
+import QuizHeader from '../components/common/QuizHeader';
+import DummyTest from '../dummytest';
```

Comment:
path alias

- apply to all files

Reply:
the path alias is implemented already on this PR sir, [[SELS-TASK]-[FE]-Mockup Results Page](https://github.com/framgia/sph-els-mark/pull/15)

Reply:
@Mjdelima20 to reduce dependency and waiting time. apply it here in the PR nalang sad.